waterlog/src/actions/CostsReportActions.js

```diff
@@ -0,0 +1,37 @@
+import {
+  FETCH_COSTS_DAILY,
+  FETCH_COSTS_MONTHLY,
+  FETCH_COSTS_SEASONALLY
+} from './Types';
+import { Globals } from './../Globals';
+
+export const fetchCostsDaily = () => (dispatch) => {
+  fetch(`${Globals.API_URL}/api/segmentevents/dailyCost`) //Change to use either localhost/server
```

Comment:
I think you can probably take these comments and put it in Globals since that would change to localhost if you're running locally.

Reply:
Removed comments and updated Global.